fili-core/src/test/java/com/yahoo/bard/webservice/druid/client/impl/AsyncDruidWebServiceImplWrapper.java

```diff
@@ -59,6 +62,7 @@ protected void sendRequest(
             final AtomicLong outstanding
     ) {
         this.request = requestBuilder.build();
+        return ConcurrentUtils.constantFuture(null); //no request to be sent
```

Comment:
Should call out what this is doing in the JavaDoc on this method since it's A-typical.